Misc/NEWS.d/next/Core and Builtins/2019-03-25-13-45-19.bpo-36440.gkvzhi.rst

```diff
@@ -0,0 +1 @@
+Include node names in ParserError messages, instead of numeric IDs
```

Comment:
Can you add formatting to the NEWS entry (so ParserError will have a link in the rendered documentation). Also, add "Patch by A. Skrobov" at the end :)